src/composables/useActiveSession.js

```diff
@@ -98,7 +98,8 @@ export function useActiveSession() {
 
 		try {
 			await setSessionState(token.value, SESSION.STATE.INACTIVE)
-			showInfo(t('spreed', 'Session has been marked as inactive'))
+			// TODO to many verbose from this state change
```

Comment:
```suggestion
			// TODO too verbose from this state change
```

Reply:
> 'showInfo' is defined but never used

Reply:
It's still WIP, so will use this atm for testing. But I'm thinking to remove it together with comment